workers/proto/kitchen_sink/kitchen_sink.proto

```diff
@@ -15,6 +15,7 @@ import "temporal/api/enums/v1/workflow.proto";
 message TestInput {
   WorkflowInput workflow_input = 1;
   ClientSequence client_sequence = 2;
+  ClientAction with_start_action = 3;
```

Comment:
I had considered using the client sequence's first item, but that seemed too surprising and messy (ie how do you represent "just start").

Reply:
I prefer it here as well